spacy_llm/tasks/rel.py

```diff
@@ -65,7 +65,7 @@ def make_rel_task(
     template: str = _DEFAULT_REL_TEMPLATE,
     label_definitions: Optional[Dict[str, str]] = None,
     examples: ExamplesConfigType = None,
-    normalizer: Optional[Callable[[str], str]] = None,
+    normalizer: Union[Callable[[str], str], str] = "spacy.LowercaseNormalizer.v1",
```

Comment:
Do we actually want to get rid of all the default values? I think usually we have default values in the factory functions, but not in the (sub-)components themselves.